Ginger/GingerCoreNET/RunLib/CLILib/CLISCM.cs

```diff
@@ -97,9 +97,12 @@ public void Execute(RunsetExecutor runsetExecutor)
         }
 
 
-        public void LoadContent(string args, CLIHelper cliHelper, RunsetExecutor runsetExecutor)
-        {
-                        
+        public void LoadGeneralConfigurations(string args, CLIHelper cliHelper)
+        {                        
+        }
+
+        public void LoadRunsetConfigurations(string content, CLIHelper cliHelper, RunsetExecutor runsetExecutor)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)